spec/features/scheduling/scheduled_publishing_spec.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+RSpec.feature "Scheduled publishing" do
+  include ActiveSupport::Testing::TimeHelpers
+
+  around do |example|
+    Sidekiq::Testing.fake! do
+      travel_to(Time.zone.parse("2019-06-21"))
+      example.run
+      travel_back
+    end
+  end
+
+  scenario do
+    given_there_is_a_schedulable_edition
+    when_i_go_to_schedule_a_publishing
+    and_i_submit_the_reviewed_option
+    then_i_see_the_edition_is_scheduled
+    and_a_publish_intent_is_created
+
+    when_the_publishing_job_runs
+    then_the_edition_is_published
+    and_the_editors_are_notified
+
+    when_i_visit_the_summary_page
+    then_i_can_see_the_publishing_succeeded
+    and_there_is_a_history_entry
```

Comment:
Minor, but I think we normally talk about 'timeline entries' in steps like this one.

Reply:
Covered in previous comment